website/docs/components/page-header/partials/code/how-to-use.md

```diff
@@ -18,6 +18,8 @@ The layout of this component is responsive and adjusts based on the width of its
 
 !!!
 
+The resonsive layout is achieved using [container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_container_queries), creating a new stacking context. We recommend setting the appropriate `z-index` value to the Page Header to make sure its content (for example a Dropdown) is not obscured by other elements on the page.
```

Comment:
Should also this live inside a "banner" (warning) box? 😬

Reply:
Yes, it's a good call – it is also very much related to the above paragraph.